src/app/components/client/dialog/Dialog.tsx

```diff
@@ -8,15 +8,19 @@ import styles from "./Dialog.module.scss";
 import { CloseBtn } from "../../server/Icons";
 import { useL10n } from "../../../hooks/l10n";
 
+type ContentAlignment = "left" | "center" | "right";
+
 export type Props = {
   children: ReactNode;
   onDismiss?: () => void;
   title?: ReactNode;
   illustration?: ReactNode;
+  contentAlignment?: ContentAlignment;
```

Comment:
I'd generally prefer to avoid too many branches inside a component, since it becomes harder to see how a change affects a special case that you're not currently triggering. (At the very least we should probably have a Storybook story with the different cases.)

I think this case in particular, it shouldn't be necessary either? Since the children of the dialog are rendered wholesale, can't we just wrap those children in an element that start-aligns its children at the call site?

Reply:
Yes, we can just handle it for the content.